client/src/components/MultiJurisdictionAudit/RoundManagement/useResults.ts

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import { api } from '../../utilities'
+
+export interface IResultValues {
+  [contestId: string]: {
+    [choiceId: string]: number | string
+  }
+}
+
+export interface IResults {
+  results: IResultValues
+}
+
+const stringifyPossibleNull = (v: string | number | null) => (v ? `${v}` : '')
+
+const reformatResults = (r: IResultValues, numberify = true): IResultValues => {
+  return Object.keys(r).reduce(
+    (a, contestId) => ({
+      ...a,
+      [contestId]: Object.keys(r[contestId]).reduce(
+        (b, choiceId) => ({
+          ...b,
+          [choiceId]: numberify
+            ? Number(r[contestId][choiceId])
+            : stringifyPossibleNull(r[contestId][choiceId]),
+        }),
+        {}
+      ),
+    }),
+    {}
+  )
+}
+
+const numberifyResults = (r: IResults): IResultValues =>
+  reformatResults(r.results)
+
+const getResults = async (
+  electionId: string,
+  jurisdictionId: string,
+  roundId: string
+): Promise<IResults | null> => {
+  try {
+    const response: IResultValues = await api(
+      `/election/${electionId}/jurisdiction/${jurisdictionId}/round/${roundId}/results`
+    )
+    return { results: reformatResults(response, false) }
```

Comment:
any reason that you're wrapping the results in `{ results: ... }` here? seems like it just makes for more unwrapping work in the component that consumes this value

Reply:
@jonahkagan It needs to be there for the form eventually, but I tried moving the wrapping and typing around a bit again and was able to localize that closer to the form itself, which did indeed clean things up more everywhere else. Thanks!